CONTRIBUTING.md

```diff
@@ -1,23 +1,36 @@
 # Contributing Guidelines
```

Comment:
Can you do the change to the CONTRIBUTING.md on another PR, you are mixing a lot of changes not related with the issue you are trying to solve, which is just about the README and not contributing guidelines. You can create an issue to improve the contributing guidelines and submit a PR for it. To help with this, its useful to create a branch per change you do, to be able to have parallel PRs with different commits associated.

Reply:
I completely agree with you Isabel about too much stuff in this PR. But what would you say if  I (only in this PR ;) ) just changed PR title to "update README and CONTRIBUTING"? 
I understand the strive for best practices but...just this small PR 🙄

Reply:
I must sound unprofessional now hahah

Reply:
> I understand the strive for best practices but...just this small PR

The "but" in that sentence... 🤣 I mean, do you want to follow best practices or nah?
Do you want to inspire the contributors to follow best practises, or will you be fine with them saying "just this small PR"?

Reply:
If ... when we reach a level that this is deployed on production, you want to uphold these standards to your contributors who devote time to help you build the app, so that we minimize confusion in the commit history, we have small Pull Requests to easily review and test, have consistency in the way we develop the project and common guidelines for everyone to follow, etc...

Reply:
Oh no. 🤦😂 i agree with isabel now.
Also i don't think we need a PR  to bump the numbers specifically, as this is a small update let contributors bump the numbers when they update the flutter code.Maybe we can add a
[ ] have you updated the packages to their latest versions from pub.dev field? 
What are your thoughts